pitest/src/test/java/org/pitest/functional/FCollectionTest.java

```diff
@@ -67,12 +66,12 @@ public void shouldApplyForEachToAllItems() {
 
   @Test
   public void shouldApplyMapToAllItems() {
-    assertEquals(this.is, FCollection.map(this.is, Prelude.id()));
+    assertEquals(this.is, FCollection.map(this.is, x -> x));
```

Comment:
There is ``java.util.function.Function.identity()`` which does exactly the same. Would have replaced Prelude.id() by this built-in function but ``x -> x`` does exactly the same. Maybe a matter of taste.... Don't feel it's worth insisting on changing that. Code is readable anyway.